paf-mvp-demo-express/src/views/advertiser/index.hbs

```diff
@@ -820,11 +820,11 @@
                   </span>
                 </a>
                 <script>
-                  document.querySelector('#preferences')
-                    .addEventListener('click', (e) => {
-                      e.preventDefault();
-                      PAF.refreshIdsAndPreferences({ proxyHostName: "{{proxyHostName}}", showPrompt: 'doPrompt'});
-                    });
+                    document.querySelector('#preferences')
+                        .addEventListener('click', (e) => {
+                            e.preventDefault();
+                            window.PAFUI.promptConsent();
```

Comment:
Actually, no, you shouldn't do this.
Seeing your change, I investigated why there was a difference between the publisher and the advertiser, and there shouldn't be.
More problematic: it happens that this "new" version (which is used on the publisher's view) has a bug.

You can test it yourself:

- access [publisher](https://www.pifdemopublisher.com/)
- the widget pops up
- close it
- click “edit marketing preferences”
- the original widget is shown (no possibility to edit id)
- click any of the two options
- nothing is saved, the user is still NOT_PARTICIPATING
- Expected behavior: should save an appropriate cookies